artifacts/attributes.adoc

```diff
@@ -6,6 +6,11 @@
 :imagesdir:
 :idseparator: -
 
+// Platforms
+:ocp-full: Red{nbsp}Hat{nbsp}OpenShift{nbsp}Container{nbsp}Platform
+:ocp: OpenShift{nbsp}Container{nbsp}Platform
+:openshift: Red{nbsp}Hat{nbsp}OpenShift
```

Comment:
https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/135 creates attributes for approved RHOCP names and applies them to the installation sections.